packages/fast-usdc/src/exos/advancer.js

```diff
@@ -161,6 +158,26 @@ export const prepareAdvancerKit = (
             const destination = chainHub.makeChainAddress(EUD);
 
             const fullAmount = toAmount(evidence.tx.amount);
+            const {
+              tx: { forwardingAddress },
+              txHash,
+            } = evidence;
+
+            const { borrowerFacet, notifyFacet, poolAccount } = this.state;
```

Comment:
sometime before this PR, but I'm surprised to see `Facet` in the name of the object. the consumer needn't be concerned with the structure in which the object resides.

I've added this to revisit in https://github.com/Agoric/agoric-sdk/issues/10432